generators/docker/multi/templates/Dockerfile.tmpl

```diff
@@ -1,6 +1,6 @@
 # This is a multi-stage Dockerfile and requires >= Docker 17.05
 # https://docs.docker.com/engine/userguide/eng-image/multistage-build/
-FROM gobuffalo/buffalo:{{.opts.Version}} as builder
+FROM gintsgints/docker-buffalo as builder
```

Comment:
??

Reply:
I propose we have different project for making docker app envionrment, as it will contain files which needed just for docker environment. As example script - wait-for.  It will have seperate versioning.
If I could create project under gobuffalo I would make that image there.

Reply:
Why `docker-buffalo` as the name...also I don't wanna rely on your image being up to date...so yeh -- let's NOT do this. I do semi-agree -- but at the same time -- it might be worth it to host the image as a buffalo repo.

Reply:
As soon as there will be some docker image which contains wait-for script, we can replace reference back. Other way my supplied docker-compose will not work at all.

Reply:
Please give me place where I could put my lovely script :)

Reply:
One more reason, to have seperate project for buffalo docker image is, we need some variations for base image, like one for arm64.